
If SDL is built before tslib, then it can't detect it at configure time and so
touchscreen's support in SDL is crappy.

Signed-off-by: Julien Boibessot <julien.boibessot@armadeus.com>

Integrated by BR for 2010.02.

Index: buildroot/package/sdl/sdl.mk
===================================================================
--- buildroot.orig/package/sdl/sdl.mk	2009-06-23 18:44:22.000000000 +0200
+++ buildroot/package/sdl/sdl.mk	2009-06-23 18:50:18.000000000 +0200
@@ -35,6 +35,11 @@
 SDL_X11=--enable-video-x11=no
 endif
 
+ifeq ($(BR2_PACKAGE_TSLIB),y)
+SDL_DEPS+=tslib
+endif
+
+
 $(DL_DIR)/$(SDL_SOURCE):
 	$(WGET) -P $(DL_DIR) $(SDL_SITE)/$(SDL_SOURCE)
 
@@ -95,7 +100,7 @@
 	cp -dpf $(STAGING_DIR)/usr/lib/libSDL*.so* $(TARGET_DIR)/usr/lib/
 	-$(STRIPCMD) $(STRIP_STRIP_UNNEEDED) $(TARGET_DIR)/usr/lib/libSDL.so
 
-SDL sdl: uclibc $(TARGET_DIR)/usr/lib/libSDL.so
+SDL sdl: uclibc $(SDL_DEPS) $(TARGET_DIR)/usr/lib/libSDL.so
 
 sdl-clean:
 	$(MAKE) DESTDIR=$(TARGET_DIR) CC=$(TARGET_CC) -C $(SDL_DIR) uninstall
